src/ray/core_worker/test/mock_worker.cc

```diff
@@ -79,6 +79,8 @@ class MockWorker {
     } else if ("MergeInputArgsAsOutput" == typed_descriptor->ModuleName()) {
       // Merge input args and write the merged content to each of return ids
       return MergeInputArgsAsOutput(args, return_ids, results);
+    } else if ("WhileTrueLoop" == typed_descriptor->ModuleName()) {
+      return WhileTrueLoop(args, return_ids, results);
```

Comment:
Will this line of code be executed?

Reply:
It should be executed when we run the TestCancelTasks C++ test (in the core_worker_test.cc)

Reply:
Hmm, `WhileTrueLoop` is a dead loop. How can the program exit?